src/main/flight/mixer.c

```diff
@@ -561,6 +578,76 @@ static void airplaneMixer(void)
 }
 #endif
 
+/*
+ * float in range [-PI/2:+PI/2]
+ */
+float getTiltAngle(void) {
+    float angleTilt = 0;
+    //get estimated position of the tilting servo
+    if (rcData[AUX1] > rxConfig->midrc) {
+        angleTilt = rcData[AUX1];
+    } else {
+        angleTilt = rcData[PITCH];
+    }
+    //find max rx range
+    float halfMaxRange = (rxConfig->maxcheck - rxConfig->mincheck) / 2.0;
```

Comment:
fixed in 2fe55cd
using rcCommand in the forst if and then using "magic number" 1000 and 2000

is there a better way? maybe there is a define somewhere? @hydra  @thenickdude